Packages/Core/Editor/Drawers/UsageIndex.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityAtoms.Editor
+{
+	public partial class AtomBaseReferenceDrawer
```

Comment:
Because the `UsageIndex` class is only appropriate for the `AtomBaseReferenceDrawer` class so instead of having that class in the same file, which in my opinion just hazes the point of the main class, I just separated them into two, but the `UsageIndex` remains as a private class inside the main one.